src/WindowsUtils/SafeHandleTest.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/base/casts.h>
+#include <gtest/gtest.h>
+
+#include "OrbitBase/ThreadUtils.h"
+#include "WindowsUtils/OpenProcess.h"
+#include "WindowsUtils/SafeHandle.h"
+
+using orbit_windows_utils::SafeHandle;
+
+TEST(SafeHandle, OwnershipTransfer) {
+  auto result = orbit_windows_utils::OpenProcess(orbit_base::GetCurrentProcessId());
+  EXPECT_TRUE(result.has_value());
```

Comment:
`ASSERT_TRUE(result.has_value());`

The access to `result.value()` below will fail - so we should not continue here.

Reply:
Done, thanks.